catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/transformer/GmdTransformer.java

```diff
@@ -75,24 +99,47 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(GmdTransformer.class);
 
-    private static final String COLON = ":";
+    public static final String GCO = "gco";
 
-    private static final GmdMetacardType GMD_METACARD_TYPE = new GmdMetacardType();
+    private static final String GML_PREFIX = "gml:";
 
-    private static final String GCO_COLON = GmdMetacardType.GCO_PREFIX + ":";
+    private static final String GCO_PREFIX = GCO + ":";
 
     private static final String TRANSFORM_EXCEPTION_MSG =
             "Unable to transform from GMD Metadata to Metacard";
 
+    private static final Pattern WHITE_SPACE_PATTER = Pattern.compile("(\\s)+");
+
+    private static final Pattern NEW_LINE_PATTERN = Pattern.compile("(\\n)+");
+
+    private static final String ZERO_HOUR_SUFFIX = "T00:00:00Z";
+
+    private static MetacardType gmdMetacardType;
+
     private final XStream xstream;
 
     private final DataHolder argumentHolder;
 
     private final XMLInputFactory xmlFactory;
 
-    public GmdTransformer() {
+    private static final ThreadLocal<WKTWriter> WKT_WRITER_THREAD_LOCAL =
```

Comment:
The main concern with using WKTWriter here is thread safety.  Added a destroy method in blueprint to avoid memory leaks.